apps/omg_rpc/lib/omg_rpc/web/plugs/health.ex

```diff
@@ -0,0 +1,135 @@
+# Copyright 2018 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.RPC.Plugs.Health do
+  @moduledoc """
+  this is primarily a Plug, but we're subscribing to Alarms as well, so that we're able to reject API calls.
```

Comment:
provide some more docs: the intent & responsibilities of the module and/or plug here